src/docs/no-more-than-1-parent-record.md

```diff
@@ -0,0 +1,55 @@
+# no-more-than-1-parent-record
+
+This rule flags queries that fetch child entities fetching more than 1 parent record with GraphQL. To resolve this error, set the parent's 'first' argument value to 1.
```

Comment:
```suggestion
Offline GraphQL does not support queries specifying more than one parent record with the 'first' argument, if that query contains child entities. To resolve this error, set the parent's 'first' argument value to 1.
```

Reply:
NB: Where is this limitation documented? I don't grok it from [Feature Limitations of Offline GraphQL
](https://developer.salesforce.com/docs/atlas.en-us.mobile_offline.meta/mobile_offline/use_graphql_limitations.htm). Cc: @haifeng-li-at-salesforce @ben-zhang-at-salesforce

Reply:
> NB: Where is this limitation documented? I don't grok it from [Feature Limitations of Offline GraphQL ](https://developer.salesforce.com/docs/atlas.en-us.mobile_offline.meta/mobile_offline/use_graphql_limitations.htm). Cc: @haifeng-li-at-salesforce @ben-zhang-at-salesforce

@khawkins I created GUS bug W-17092009 to add the limitation to the Feature Limitations of Offline GraphQL topic.